tools/react-app-template/src/App.tsx

```diff
@@ -0,0 +1,65 @@
+// (C) 2019-2022 GoodData Corporation
+import React from "react";
+import { BackendProvider, WorkspaceProvider } from "@gooddata/sdk-ui";
+import { Dashboard } from "@gooddata/sdk-ui-dashboard";
+
+import { backend } from "./backend";
+import { workspace } from "./constants";
+import logo from "./assets/gooddata-logo.svg";
+import * as Md from "./md/full";
+
+const Content: React.FC = () => {
+    return (
+        <div className="container">
+            <div className="header">
+                <div className="logo">
+                    <a className="logo-link" href="https://sdk.gooddata.com/gooddata-ui">
+                        <img className="logo-link-image" src={logo} />
+                        <h1 className="logo-link-title">GoodData.UI</h1>
+                    </a>
+                </div>
+                <div className="navigation">
+                    <a className="navigation-link" href="https://sdk.gooddata.com/gooddata-ui" target="blank">
+                        📚 Official documentation
+                    </a>
+                    <a
+                        className="navigation-link"
+                        href="https://sdk.gooddata.com/gooddata-ui/docs/quickstart.html"
+                        target="blank"
```

Comment:
Should probably be `target="_blank"` no (same in the other links in this file)? Also please add `rel="noopener noreferrer"` for links opened in a new tab.

Reply:
Updated